### PR TITLE
feat: migrate slider

### PR DESCRIPTION
**Pull request by guillaumewttj**
## COMPATIBILITY

- [ ] Tested on Safari (desktop)
- [ ] Tested on Chrome (desktop)
- [ ] Tested on Firefox (desktop)
- [ ] Tested on mobile device sizes
- [ ] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [ ] Thoroughly tested in local environment
- [ ] Added tests for all new features
- [ ] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/Slider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+                type="number"
+                value={inputValue.toString()}
+              />
+            ) : (
+              <div>{min}</div>
+            ))}
+
+          <div className={cx('slider-wrapper')}>
+            <input
+              className={cx('input-range', 'slider-input-range')}
```
Comment (Isaacmaamouche) [inline lib/src/components/Slider/index.tsx]: Can we merge this 2 classes ?

Comment (guillaumewttj) [reply lib/src/components/Slider/index.tsx]: i've created those two because we share most of the css with the Range component (via input-range) and then all the slider-specific css is in `slider-input-range`

Comment (guillaumewttj) [reply lib/src/components/Slider/index.tsx]: those components are pretty ugly in terms of code in general, i think we need a more global refactoring on them

File: lib/src/components/Slider/Range.tsx [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+}) => {
+  let ensuredValue = Math.max(value, toCompare + 1 * step)
+  ensuredValue = Math.min(ensuredValue, max)
+  return round(ensuredValue, step)
+}
+
+/**
+ * @name Slider.Range
+ */
+// BREAKING CHANGE: removed `borderSelectorColor` default value
```
Comment (cnairi) [inline lib/src/components/Slider/Range.tsx]: Could you add this to `welcome-ui/website/build-app/pages/foundations/upgrades/v9.mdx` please?

File: lib/src/components/Slider/docs/examples/border-selector.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        max={100}
+        min={0}
+        onChange={handleChangeRange}
+        value={{ max: 75, min: 25 }}
+      />
+    </>
+  )
+}
+
+export default Example
```
Comment (cnairi) [inline lib/src/components/Slider/docs/examples/border-selector.tsx:33]: Not sure this example is still relevant? As you marked the borderSelectorColor as deprecated?

**Review by cnairi — COMMENTED** [review]
Hi @guillaumewttj 

I also noticed that the slider range are not working in the documentation: https://wui-170-slider.preview.welcome-ui.com/components/slider

(I cannot slide when it's a range)

+ the cursor are blue
Capture d’écran 2025-09-29 à 12 36 14

File: lib/src/components/Slider/Range.tsx (hunk 2 above)
Comment (cnairi) [reply lib/src/components/Slider/Range.tsx]: (removed, as I implemented a default value in our scss)

File: lib/src/components/InputText/input-text.module.scss [hunk 4]
```diff
@@ -1,7 +1,6 @@
 @layer components {
   .input-text-wrapper {
     position: relative;
-    width: 100%;
```
Comment (Leiksa) [inline lib/src/components/InputText/input-text.module.scss:4]: Maybe we could expose and override a custom prop (`--inputWrapperWidth`) for the slider because this prevents inputs from having a width of 100%
Screenshot 2025-09-30 at 10 39 49

Comment (cnairi) [reply lib/src/components/InputText/input-text.module.scss:4]: Thanks 🙏 [3c1e247](https://github.com/WTTJ/welcome-ui/pull/2922/commits/3c1e247b73cc3187169c72178f81b4fa17220358)

**Review by Leiksa — APPROVED** [review]
